--- ./crates/compat/Cargo.toml.orig	2024-06-20 12:18:01.431793227 +0200
+++ ./crates/compat/Cargo.toml	2024-06-20 12:36:40.794150064 +0200
@@ -2,12 +2,11 @@
 name = "wasmcloud-compat"
 version = "0.1.0"
 description = "pre-WIT functionality for post-WIT world"
-
-authors.workspace = true
-categories.workspace = true
-edition.workspace = true
-license.workspace = true
-repository.workspace = true
+authors = ["The wasmCloud Team"]
+categories = ["wasm"]
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/wasmCloud/wasmCloud"
 
 [dependencies]
 anyhow = { workspace = true, features = ["std"] }
