--- ./crates/provider-archive/Cargo.toml.orig	2024-06-20 12:18:01.434793237 +0200
+++ ./crates/provider-archive/Cargo.toml	2024-06-20 12:40:32.388099570 +0200
@@ -6,11 +6,10 @@
 readme = "README.md"
 keywords = ["jwt", "zip", "archive", "security", "wasmcloud"]
 categories = ["cryptography", "authentication", "wasm"]
-
-authors.workspace = true
-edition.workspace = true
-license.workspace = true
-repository.workspace = true
+authors = ["The wasmCloud Team"]
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/wasmCloud/wasmCloud"
 
 [dependencies]
 async-compression = { workspace = true, features = ["tokio", "gzip"] }
