--- ./crates/host/Cargo.toml.orig	2024-06-20 12:18:01.432793230 +0200
+++ ./crates/host/Cargo.toml	2024-06-20 13:12:11.721907739 +0200
@@ -2,12 +2,11 @@
 name = "wasmcloud-host"
 version = "0.82.0"
 description = "wasmCloud host library"
-
-authors.workspace = true
-categories.workspace = true
-edition.workspace = true
-license.workspace = true
-repository.workspace = true
+authors = ["The wasmCloud Team"]
+categories = ["wasm"]
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/wasmCloud/wasmCloud"
 
 [badges.maintenance]
 status = "actively-developed"
@@ -42,8 +41,8 @@
 url = { workspace = true, features = ["serde"] }
 uuid = { workspace = true, features = ["serde"] }
 wascap = { workspace = true }
-wasmcloud-compat = { workspace = true }
+wasmcloud-compat = { workspace = true, path = "../compat" }
 wasmcloud-control-interface = { workspace = true }
 wasmcloud-core = { workspace = true, features = ["otel"] }
-wasmcloud-runtime = { workspace = true }
+wasmcloud-runtime = { workspace = true, path = "../runtime" }
 wasmcloud-tracing = { workspace = true, features = ["otel"] }
